TACC/protx
# Task/predictive features cosmetic fixes

## Overview: ##

A quick PR to fix some cosmetic label issues on the predictive charts table and to cleanup some lingering CSS linting issues. 

## Related Jira tickets: ##

* No ticket, quick task.

## Summary of Changes: ##

## Testing Steps: ##

1. Run project, make sure everything looks the same.
2. See a new chart title for Predictive Features.

## UI Photos:

<img width="886" alt="Screen Shot 2021-08-23 at 4 52 41 PM" src="https://user-images.githubusercontent.com/3238078/130524401-97c6960f-6d4e-430d-aabe-2bf02388e872.png">


## Notes: ##
